src/app/config/or_options_st.h

```diff
@@ -634,6 +634,12 @@ struct or_options_t {
   /** If true, include statistics file contents in extra-info documents. */
   int ExtraInfoStatistics;
 
+  /** If true, activate reporting of all bandwidth statistics.
+   *  Defaults to auto which let's the consensus decide the reporting part.
```

Comment:
We don't list defaults in comments, because they can change.
```suggestion
   *  If auto, check the BandwidthStatistics consensus parameter.
```